### PR TITLE
[homematic] Update README.md

### PR DESCRIPTION
**Pull request by MeisterPiffer**
I came over this topic: https://community.openhab.org/t/refreshtypes-not-working-with-oh-3/114715/4 as I tried to follow the docs, which didn't mention rules DSL and I think this should be in the docs as I guess many people use Rules DSL. Hence adding another example.
Not sure at all, if I'm doing it the right way. This is my first try to contribute something via github. Thanks

Ok, now that I read the links below and contributing guidelines, I'm 100% sure that I did something wrong, but I'll try anyway :)

### CODE REVIEW

**Comment by jlaur** [comment]
@MeisterPiffer - you could have a look at this:
https://github.com/openhab/openhab-addons/blob/7855f81760ae6625b974f5d9147344026aa1b339/bundles/org.openhab.binding.omnilink/README.md#L146-L192

And how it is rendered:
https://www.openhab.org/addons/bindings/omnilink/#rule-actions

**Comment by MeisterPiffer** [comment]
Thanks a lot!
I tried to change this, but I'm not really familiar with pull requests in github. I used "edit file" and it looks like this is an update to the request.

File: bundles/org.openhab.binding.homematic/README.md [hunk 1]
```diff
@@ -660,11 +660,18 @@ openhab:send Var_1 REFRESH
 
 In scripts:
 
-```javascript
+:::: tabs
+
+::: tab JavaScript
 import org.openhab.core.types.RefreshType
```
Comment (jlaur) [inline bundles/org.openhab.binding.homematic/README.md:666]: It seems like you accidentally lost the JavaScript code:
```suggestion
import org.openhab.core.types.RefreshType

Var_1.sendCommand(RefreshType.REFRESH)
```

File: bundles/org.openhab.binding.homematic/README.md [hunk 2]
```diff
@@ -660,11 +660,18 @@ openhab:send Var_1 REFRESH
...
-...
-Var_1.sendCommand(RefreshType.REFRESH)
-```
+:::
+
+::: tab DSL
+
```
Comment (jlaur) [inline bundles/org.openhab.binding.homematic/README.md:670]: Remove this line feed.

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks for your contribution, please see my attached comments.

**Comment by jlaur** [comment]
> I tried to change this, but I'm not really familiar with pull requests in github. I used "edit file" and it looks like this is an update to the request.

You are doing fine. The new changes were provided as a new commit, so everything is correct. :)

File: bundles/org.openhab.binding.homematic/README.md (hunk 2 above)
Comment (MeisterPiffer) [reply bundles/org.openhab.binding.homematic/README.md:672]: I can't see how to do that. It seems to me as you suggested this, but how can I confirm this? Do I just hit "Resolve conversation"?

Comment (jlaur) [reply bundles/org.openhab.binding.homematic/README.md:672]: You can either edit again and delete the empty line, or you can apply the suggested fix. After this you can mark the comment as resolved.

**Review by jlaur — APPROVED** [review]
LGTM

**Comment by MeisterPiffer** [comment]
> @MeisterPiffer - today it's possible to see how the new version of the documentation is rendered: https://next.openhab.org/addons/bindings/homematic/#troubleshooting
> 
> It seems that we missed one important thing: Within the tabs the code must still be within code fences like shown in the example.

Yes, I totally missed that. I'll file another pull request, to get some practice :) Thank you very much!

**Pull request merged**